rdkit/ML/Scoring/UnitTestScoring.py

```diff
@@ -147,6 +147,38 @@ def test4_BEDROC(self):
     bedroc = Scoring.CalcBEDROC(self.scoreAllDecoys, self.index, self.alpha)
     self.assertEqual(bedroc, 0.0)
 
+  def test_safeguard_sort(self):
+
+    # Garbage scores with sorted classes show garbage performance
```

Comment:
It is garbage there because I'm using nans in that example. But I will remove words like rotten and garbage that look bad in a decent codebase.